src/librustc/session/mod.rs

```diff
@@ -431,8 +431,13 @@ impl Session {
                     diag_builder.span_note(span, message);
                 }
                 DiagnosticBuilderMethod::SpanSuggestion(suggestion) => {
-                    let span = span_maybe.expect("span_suggestion needs a span");
-                    diag_builder.span_suggestion(span, message, suggestion);
+                    let span = span_maybe.expect("span_suggestion_* needs a span");
+                    diag_builder.span_suggestion_with_applicability(
+                                                                    span,
+                                                                    message,
+                                                                    suggestion,
+                                                                    Applicability::Unspecified,
+                                                                    );
```

Comment:
Ah, I see what the formatting commits were about now: Please, for all the people who like split diffs, indent the arguments just one level and not visually :)

Reply:
@vi the only thing left is the formatting here (and elsewhere) should have the arguments indented one level (four spaces):

```
                    diag_builder.span_suggestion_with_applicability(
                        span,
                        message,
                        suggestion,
                        Applicability::Unspecified,
                    );
```